tests/integrations/config/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+options:
+  no_parent_owners: true
+filters:
+  "(OWNERS|test-custom\\.toml)$":
```

Comment:
Can you skip this file? It's a test-only file, and it's a bit overkill to seek 'sig-critical-approvers-config-src' permission to add tests.